.github/workflows/upgrade-test.yaml

```diff
@@ -0,0 +1,52 @@
+name: Upgrade Tests
```

Comment:
Why can't this be part of integration test itself? If naming is the issue, change the name.